### DISCUSSION TITLE
Feature request; add e-mail address for main page

### DISCUSSION THREAD

**Discussion by DMHas** [keepassxreboot/keepassxc#8716] [Ideas]
Hello KeePassXC Team,

KeePassXC is a great program. But i miss the option to show the E-mail address for the entries at main page and preview area. Is there a chance to add e-mail address to this screens areas?

Thank you in advance!

Sincerely,
DMHas

**Comment by droidmonkey** [comment]
It is planned for 2.8.0

**Comment by DMHas** [comment]
@droidmonkey: Thank you much for your quick reply! That's good news! Do you can estimate, when V2.8.0 will be released?
